source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb

```diff
@@ -756,14 +756,34 @@
         </table>
       </td>
     </tr>
+    <% ev_networks = MultiJson.load(RestClient.get("https://developer.nrel.gov/api/alt-fuel-stations/v1/electric-networks.json", :params => { :country => "all", :api_key => ENV["DOCS_API_KEY"] })) %>
     <tr>
       <th class="doc-parameter-name" scope="row">ev_network</th>
       <td class="doc-parameter-value">
         <div class="doc-parameter-value-field">
           <strong>Type:</strong> string
         </div>
       </td>
-      <td class="doc-parameter-description">For electric stations, the name of the EV charging network, if applicable.</td>
+      <td class="doc-parameter-description">
+        <p>For electric stations, the name of the EV charging network, if applicable.</p>
+        <table border="0" cellpadding="0" cellspacing="0" class="table table-bordered">
+          <thead>
+            <tr>
+              <th scope="col">Option</th>
```

Comment:
Thanks for adding this. Could you change "Option" to "Value", though? I think that's more consistent with the rest of the tables under Response Fields.

Reply:
fixed